### ISSUE TITLE
DataHub provides test and learn support to Gensyn

### ISSUE THREAD

**Issue by mcleo-d** [finos/datahub#32]
### Description

cc @benfielding from Gensyn.

This issue has been created to help support and guide Gensyn in the **Installation**, **Running** and **Modelling** of test data to inform Gensyn feedback and next steps into the DataHub project team.

### Success Criteria

- [x] Gensyn can `Clone`, `Build` and `Run` DataHub on a local machine. 
- [x] Gensyn has modelled test data that represents a small scale internal test scenario.
- [x] Gensyn has provided feedback and next steps to the DataHub project team.

### Definition of Done

Gensyn have enough information to make an informed choice on moving forward with DataHub and have provided collaboration requirements to the DataHub project team whilst agreeing next steps.

**Comment by mcleo-d** [comment]
Also related to https://github.com/finos/datahelix/issues/1684

**Comment by BenFielding** [comment]
Thanks James, will dive in and let you know how it goes.

**Comment by mcleo-d** [comment]
Hi @BenFielding 

Earlier this month we met to discuss FINOS DataHub and DataHelix in relation to Gensyn synthetic data requirements. 

Since our last meeting the FINOS synthetic data teams have been working with stakeholders from the open source community to gather requirements so we can shape our product roadmaps accordingly.

The reason for my email is to ask ...

  1. Did you have the opportunity to discuss evaluating DataHub and DataHelix with the rest of the team as a result of our discussion?
  2. Would you be willing to share your synthetic data requirements with the DataHub and DataHelix teams so we can incorporate your requirements into our product roadmaps? 

Finally, but not necessary, would you be willing to join combined DataHub and DataHelix project sessions to help prioritise and groom our backlogs to keep us moving forward?

Thanks for your time and I look forward to hearing from you soon.

@mcleo-d

**Comment by BenFielding** [comment]
Hi @mcleo-d,

Thanks for the chat with yourself and @grovesy yesterday, where we covered the points in your latest comment.

In terms of this issue:-

- [X] Gensyn can `Clone`, `Build` and `Run` DataHub on a local machine.
- [X] Gensyn has modelled test data that represents a small scale internal test scenario.
- [X] Gensyn has provided feedback and next steps to the DataHub project team.

For the latter two points - we don't have a specific use case within Gensyn for Datahub as we're a technology partner working on contributing to the direction of the project and developing similar technologies commercially. We have tested Datahub with the given examples and understand the performance of the library. We're in the process of providing feedback and next steps through contribution to #30 and expect this to be an ongoing process.

Looking forward to meeting regularly.
